src/status_im/ui2/screens/chat/group_details/view.cljs

```diff
@@ -66,44 +66,41 @@
      :weight :medium
      :style  {:color (colors/theme-colors colors/neutral-50 colors/neutral-40)}} title]])
 
-(def added (reagent/atom ()))
-
-(defn contact-requests-sheet
-  [group]
-  (let [added (reagent/atom ())]
-    (fn []
-      [:f>
-       (fn []
-         (let [{window-height :height} (rn/use-window-dimensions)
-               safe-area               (safe-area/use-safe-area)]
-           [rn/view {:style {:height (- window-height (:top safe-area))}}
-            [rn/touchable-opacity
-             {:on-press #(rf/dispatch [:bottom-sheet/hide])
-              :style    (style/close-icon)}
-             [quo2/icon :i/close {:color (colors/theme-colors colors/neutral-100 colors/white)}]]
-            [quo2/text
-             {:size   :heading-1
-              :weight :semi-bold
-              :style  {:margin-left 20}}
-             (i18n/label :t/add-members)]
-            [rn/text-input
-             {:placeholder (str (i18n/label :t/search) "...")
-              :style       {:height             32
-                            :padding-horizontal 20
-                            :margin-vertical    12}
-              :on-change   (fn [e]
-                             (rf/dispatch [:contacts/set-search-query (oget e "nativeEvent.text")]))}]
-            [contact-list/contact-list
-             {:icon    :check
-              :group   group
-              :added   added
-              :search? true}]
-            [rn/view {:style style/bottom-container}
-             [quo2/button
-              {:style    {:flex 1}
-               :on-press #(rf/dispatch [:bottom-sheet/hide])
-               :disabled (zero? (count @added))}
-              (i18n/label :t/save)]]]))])))
+(defn add-members-sheet
+  [group admin?]
+  [:f>
+   (fn []
+     (let [{window-height :height} (rn/use-window-dimensions)
+           safe-area               (safe-area/use-safe-area)
+           selected-participants   (rf/sub [:group-chat/selected-participants])
+           deselected-members      (rf/sub [:group-chat/deselected-members])]
+       [rn/view {:style {:height (- window-height (:top safe-area))}}
```

Comment:
something wrong is here ,we shouldn't use height ,  and safe area is applied to all screens by default , so we shouldn't care about them in the screen itself

Reply:
That's a bottom sheet not a screen

Reply:
mhm